### ISSUE TITLE
Tourniquet

### ISSUE THREAD

**Issue by Pyrodes** [studio-minus/ppg-bugs-and-requests#1500]
**Feature description**
Adds a new wire called a tourniquet which, when applied to a human, stops bleeding on and below the limb it's applied to at the cost of slowly rotting the affected limbs due to lack of blood flow. Does not work on the torso/head.

**Feature purpose**
This feature would be useful/practical for when you don't have time to apply bandages on multiple limbs or don't want to go through the trouble of bandaging each limb that is bleeding.

**Additional context**
![image](https://user-images.githubusercontent.com/68454411/174416286-60ad2473-1c39-47de-8a35-eb1bf875d58c.png)

**Comment by DindinYT37** [comment]
I'm guessing this would work similar to the phase link, where you would have to "select" two limbs and it would only work on connected limbs?

**Comment by Pyrodes** [comment]
> I'm guessing this would work similar to the phase link, where you would have to "select" two limbs and it would only work on connected limbs?

I suppose it could work that way, but I originally intended for it to be applied on an upper limb, then affect the limb below it although unsure how hard that would be to code in.

**Comment by DindinYT37** [comment]
> > I'm guessing this would work similar to the phase link, where you would have to "select" two limbs and it would only work on connected limbs?
> 
> I suppose it could work that way, but I originally intended for it to be applied on an upper limb, then affect the limb below it although unsure how hard that would be to code in.

I don't see why it should only be restricted to an upper limb though, it would be funnier if you could also use it on things like the head lmao.

**Comment by bywok** [comment]
Bro there's already bandages that do this... hover over the Springs button

**Comment by Pyrodes** [comment]
> I don't see why it should only be restricted to an upper limb though, it would be funnier if you could also use it on things like the head lmao.

Yeah lol I'm in favor of no restrictions, but I just don't realistically see it fulfilling its intended purpose that way without killing the human of course so maybe a different effect.

**Comment by DindinYT37** [comment]
> > I don't see why it should only be restricted to an upper limb though, it would be funnier if you could also use it on things like the head lmao.
> 
> Yeah lol I'm in favor of no restrictions, but I just don't realistically see it fulfilling its intended purpose that way without killing the human of course so maybe a different effect.

I mean, you would still be able to do what you originally suggested, it's just that you can also - for example, suffocate the human by putting one between the uppertorso and the head.

**Comment by Pyrodes** [comment]
> I mean, you would still be able to do what you originally suggested, it's just that you can also - for example, suffocate the human by putting one between the uppertorso and the head.

Yeah, I suppose that would make sense.

**Comment by Cornbreadbugaloo** [comment]
this would be so epic
